### ISSUE TITLE
3.4.02: Not closed comment sign, resulting is not executing a statement?

### ISSUE THREAD

**Issue by albert-github** [samboy/MaraDNS#64]
In the file: https://github.com/samboy/MaraDNS/blob/master/sqa/asktest.c#L63 we see the code:
``` 
    /* Set the plaintext block to 'zzzzzzzzz...' /
    memset(r_inBlock,'z',16);
    /* If the hostname is longer than 16 characters, make the plaintext
```
(line 63 ff.), where at the first line it looks like an `*` is missing before the "closing `/` and thus `memset...` is not executed!

Is this intentionally or by accident?
When intentionally make the comment block more clear, i.e explicitly comment out the `memset` like and do not use the side effect that the block might be closed ly the next comment
```
    /* Set the plaintext block to 'zzzzzzzzz...' /
    memset(r_inBlock,'z',16); */
```
otherwise, please fix.

**Comment by samboy** [comment]
I haven’t used that code in well over a decade; it’s part of a SQA test I no longer run. That probably *was* a typo; I’ll fix it for 3.4.02.

**Comment by samboy** [comment]
I have updated the two “sources of truth” to fix this bug:

* The git tree
* The tarball for the next version at http://maradns.samiam.org/download/3.4/snap/

Closing.
